app/routes/thank-you/route.tsx

```diff
@@ -1,3 +1,31 @@
+import { useLoaderData, json } from '@remix-run/react';
+import type { LoaderFunctionArgs } from '@remix-run/node';
+import { getEcomApi } from '~/api/ecom-api';
+import { CategoryLink } from '~/components/category-link/category-link';
+import { OrderSummary } from '~/components/order-summary/order-summary';
+import styles from './thank-you.module.scss';
+
+export const loader = async ({ request }: LoaderFunctionArgs) => {
+    const orderId = new URL(request.url).searchParams.get('orderId');
+    if (!orderId) throw new Error('Order not found');
+    const order = await getEcomApi().getOrder(orderId);
+    return json({ order });
```

Comment:
Do we need `json`?

Reply:
Yes. I import it from `@remix-run/react` now, not `@remix-run/node`, so there shouldn't be a problem.
> This is a shortcut for creating `application/json` responses. Converts `data` to JSON and sets the `Content-Type` header.

Reply:
@Letushev anyway it's better to check it in Codux